README.wine.md

```diff
@@ -0,0 +1,38 @@
+For the development of HarfBuzz, the Microsoft shaping technology, Uniscribe,
+as a widely used and tested shaper is used as more-or-less opentype reference
+implemenetation and that specially is important where opentype specification
+is or wasn't that clear. For having access to Uniscribe on Linux/macOS these
+steps are recommended:
+
+1. Install Wine from your favorite package manager.
+
+2. And `mingw-w64` compiler.
+   With `brew` on macOS, you can have it like `brew install mingw-w64`
+
+3. Download and put [this](https://drive.google.com/open?id=0B3_fQkxDZZXXbWltRGd5bjVrUDQ)
+   on your `~/.local/i686-w64-mingw32`.
+
+4. Replace all the instances of `/home/behdad/.local/i586-mingw32msvc`
```

Comment:
Just give me the fixed zip file and I'll upload it again.

Reply:
After removing the build folder, I found I was wrong.